engine/execution/computation/execution_verification_test.go

```diff
@@ -35,8 +37,14 @@ var chain = flow.Mainnet.Chain()
 
 func Test_ExecutionMatchesVerification(t *testing.T) {
 
+	noTxFee, err := cadence.NewUFix64("0.0")
+	require.NoError(t, err)
+
 	t.Run("empty block", func(t *testing.T) {
-		executeBlockAndVerify(t, [][]*flow.TransactionBody{})
+		executeBlockAndVerify(t,
```

Comment:
We need to change this to some standard execution FVM as we once discussed, but it's OK for now